packages/xo-server-transport-icinga2/src/index.js

```diff
@@ -0,0 +1,126 @@
+import createLogger from '@xen-orchestra/log'
+import httpRequestPlus from 'http-request-plus'
+
+// =============================================================================
+
+const log = createLogger('xo:xo-server:transport-icinga2')
+
+// =============================================================================
+
+export const configurationSchema = {
+  type: 'object',
+
+  properties: {
+    server: {
+      type: 'string',
+      description: 'The icinga2 server address',
+    },
+    port: {
+      type: 'integer',
+      description: 'The access port',
+    },
+    user: {
+      type: 'string',
+      description: 'The icinga2 server username',
+    },
+    password: {
+      type: 'string',
+      description: 'The icinga2 server password',
+    },
+    filter: {
+      type: 'string',
+      description: 'The filter to use',
+    },
+    rejectUnauthorized: {
+      type: 'boolean',
+      description: 'Reject unauthorized certificates',
+      default: true,
```

Comment:
IMHO, default boolean values should almost always be `false`, and *reject unauthorized* is difficult to understand because it's kind of a double negative.

Reversing the test might make things clearer.

Reply:
agreed